_test/test_mpi_wrappers/test_MESS_NAMES_factory.m

```diff
@@ -12,75 +10,106 @@
             end
             obj = obj@TestCase(name);
         end
-        function test_persistent(obj)
+        function test_get_mess_class(~)
```

Comment:
There should really be tests for every message type name -> class mapping, and tests for each of the failure modes

Reply:
yes --- this left for the more tests tickets. This one needs big bunch of additional tests, including tests for all kinds of input to the methods.